src/main/js/controls/Graph/Graph.js

```diff
@@ -320,10 +320,13 @@ class Graph extends Construct {
      *  @returns {Graph} - Graph instance
      */
     resize() {
-        setCanvasWidth(this.graphContainer, this.config);
-        scaleGraph(this.scale, this.config);
-        translateGraph(this);
-        this.content.forEach((control) => control.resize(this));
+        // Check if graphContainer is present and then resize the graph
+        if (this.graphContainer) {
+            setCanvasWidth(this.graphContainer, this.config);
+            scaleGraph(this.scale, this.config);
+            translateGraph(this);
+            this.content.forEach((control) => control.resize(this));
+        }
```

Comment:
No unit tests for this change?

Reply:
Added unit test [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/f7635b1fd30a83f25a8a30fc27a7829a70e4e0f1).